src/main.ts

```diff
@@ -40,10 +40,6 @@ import utils = require("./utils");
 // tslint:disable-next-line: no-var-requires
 const PackageJSON: any = require("../../package.json");
 
-// NOTE: We will need to find a better way to deal with the required
-//       PS Editor Services version...
-const requiredEditorServicesVersion = "2.0.0";
```

Comment:
This isn't used anywhere, right?

Reply:
if it was by Typescript, then the build would've been green. It was indirectly used by the PowerShell release function that I removed in this PR as well. I believe this is an old artefact from the days when the preview was using PSES v2 but  the RTM version was still on PSES v1. I cc'd @rjmholt  to confirm this though

Reply:
I think you're right @bergmeister